packages/@vue/babel-preset-app/README.md

```diff
@@ -16,6 +16,8 @@ This is the default Babel preset used in all Vue CLI projects. **Note: this pres
   - set to `{ node: 'current' }` when running unit tests in Node.js
 - Includes `Promise` polyfill by default so that they are usable even in non-transpiled dependencies (only for environments that need it)
 
+Sidenote, by default `@babel/preset-env` will use [`browserslist config sources`](https://github.com/browserslist/browserslist#queries) (browserslist key in package.json file is recommend) unless either the [`targets`](https://babeljs.io/docs/en/babel-preset-env#targets) or [`ignoreBrowserslistConfig`](https://babeljs.io/docs/en/babel-preset-env#ignorebrowserslistconfig) options are set.
```

Comment:
I suggest that we merge this sidenote into the first bullet point of the `targets` block as that makes the purpose of this note clearer.

Reply:
+1